examples/benchmark/tls_bench.c

```diff
@@ -1186,7 +1199,7 @@ static void* client_thread(void* args)
     ret = bench_tls_client(info);
 
     pthread_cond_signal(&info->to_server.cond);
-    info->to_client.done = 1;
+    info->to_server.done = 1;
```

Comment:
Please revert this line. The way it was previously is valid. My recommend for this fix was not right.